mllib/src/main/scala/org/apache/spark/mllib/classification/NaiveBayes.scala

```diff
@@ -446,8 +460,14 @@ object NaiveBayes {
   /** String name for Bernoulli model type. */
   private[spark] val Bernoulli: String = "bernoulli"
 
+  /** String name for complementary multinomial model type. */
+  private[spark] val Complementary_Multinomial: String = "complementary_multinomial"
```

Comment:
I don't think these should be unrelated types. Complementary-ness is orthogonal to binary vs multinomial. The constant name is not conventional in any event.